VSTSDropProxy.cs

```diff
@@ -29,47 +29,49 @@ public class VSTSDropProxy
 
         private readonly Uri _VSTSDropUri;
         private readonly string _relativeroot;
-        
-        private readonly ISet<VstsFile> _files;
-        
+
+        private readonly IList<VstsFile> _files;
+
         public VSTSDropProxy(string VSTSDropUri, string path, string pat, TimeSpan blobtimeout)
         {
-            
             _dropApi = new RestfulDropApi(pat);
             _contentClient = new HttpClient() { Timeout = blobtimeout };
-              
+
             if (!Uri.TryCreate(VSTSDropUri, UriKind.Absolute, out _VSTSDropUri))
             {
                 throw new ArgumentException($"VSTS drop URI invalid {VSTSDropUri}", nameof(VSTSDropUri));
             }
-            
-            
-            
+
             if (path == null)
             {
                 throw new ArgumentException($"VSTS drop URI must contain a ?root= querystring {_VSTSDropUri}", nameof(VSTSDropUri));
             }
-            
-            _relativeroot = path.Replace("\\", "/");
+
+            _relativeroot = path.Replace('\\', Path.DirectorySeparatorChar);
```

Comment:
I'd leave it as linux only for now and keep /

Reply:
I think it's more self documenting to use Path.DirectorySeparatorChar (which is '/' anyways).